packages/xo-web/src/common/xo/index.js

```diff
@@ -497,9 +497,11 @@ export const subscribeHostMissingPatches = (host, cb) => {
   const hostId = resolveId(host)
 
   if (missingPatchesByHost[hostId] == null) {
-    missingPatchesByHost[hostId] = createSubscription(() =>
-      getHostMissingPatches(host)
-    )
+    host.power_state !== 'Running'
+      ? (missingPatchesByHost[hostId] = () => [])
+      : (missingPatchesByHost[hostId] = createSubscription(() =>
+          getHostMissingPatches(host)
```

Comment:
Do we want to handle the case when the host shut down after the creation of the subscription?